starter-code/index.html

```diff
@@ -0,0 +1,178 @@
+                <a class="nav-link" href="#">Link</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">Link</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link disabled" href="#">Disabled</a>
+            </li>
+        </ul>
+    </div> 
+    <div class="container-fluid">
+        <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
+            <ol class="carousel-indicators">
+                <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
+                <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
+                <li data-target="#carouselExampleIndicators" data-slide-to="2"></li>
+            </ol>
+            <div class="carousel-inner">
+                <div class="carousel-item active">
+                <img class="d-block w-100" src="img/slider-1.jpg" alt="First slide">
+                    <div class="carousel-caption d-none d-md-block">
+                        <h3>What is your cat revealing to you when she purrs? What is your dog expressing when he yawns or wags his tail?</h3>
+                        <p>Understanding your cat and dog’s behaviour and the way they communicate with you, will enable you to better understand their needs and strengthen your relationship with them.</p>
+                    </div>
+                </div>
+                <div class="carousel-item">
+                <img class="d-block w-100" src="img/slider-2.jpg" alt="Second slide">
+                <div class="carousel-caption d-none d-md-block">
+                        <h3>Coffee is a major export commodity: it is the top agricultural export for numerous countries and is among the world's largest legal agricultural exports.</h3>
+                        <p>It is one of the most valuable commodities exported by developing countries. Green (unroasted) coffee is one of the most traded agricultural commodities in the world.</p>
+                    </div>
+                </div>
+                <div class="carousel-item">
+                <img class="d-block w-100" src="img/slider-3.jpg" alt="Third slide">
+                <div class="carousel-caption d-none d-md-block">
+                        <h3>Papaver is a genus of 70–100 species of frost-tolerant annuals, biennials, and perennials native to temperate and cold regions of Eurasia, Africa and North America. It is the type genus of the poppy family, Papaveraceae.</h3>
+                        <p>The flowers have two sepals that fall off as the bud opens, and four (or up to six) petals in red, pink, orange, yellow, or lilac. There are many stamens in several whorls around a compound pistil, which results from the fusion of carpels. </p>
+                    </div>
+                </div>
+            </div>
+            <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
```

Comment:
Arrows are not working for me. Bootstrap is also using jQuery, so you may have something broken related to this jquery.